src/app-bundles/locations-map-bundle.js

```diff
@@ -15,6 +16,7 @@ export default {
   getReducer: () => {
     const initialData = {
       mapState: {},
+      mapZoom: {},
```

Comment:
the mapState is supposed to store the zoom, center, and any other map properties that we want to save. I think we should use that instead of redefining the mapZoom